src/commands/buildCommands/trn.js

```diff
@@ -75,10 +75,16 @@ const componentTrnDefinitions$ = localTrns$.map(trnsFromFile => ({
 
 // Function for importing Flatpickr (fp) locale data for a particular 2-character
 // language code code - see https://flatpickr.js.org/localization/
-const fpLocale = lang =>
-	require(require.resolve(`flatpickr/dist/l10n/${lang}`, {
+const fpLocale = lang => {
+	const langFile = require(require.resolve(`flatpickr/dist/l10n/${lang}`, {
 		paths: [paths.repoRoot],
-	}))[lang];
+	}));
+	if (!langFile)
```

Comment:
ah, such a nitpick, sorry, but can we put curly braces around the if block please? :) otherwise, looks good!